general/README.md

```diff
@@ -0,0 +1,12 @@
+# General Guidelines
+
+## Principles
+
+- Don't try to reinvent the wheel, use and contribute to the open source
+- Code is read much more often than it is written, exercising clean code matters to us
+- Refactoring and unit testing is part of the job, do it when needed
+- Optimize code for people first, beware of premature optimization
+- Use tools to enforce code style and good practices
+- Clean up unused code and don't design ahead
+- Keep documentation close to the code
```

Comment:
Keep documentation close to the code. Best documentation is good name

Reply:
Alebo vseobecnejsie "Best documentation is self-explaining code"?

Reply:
alebo ;-)